index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css"/>
+    <title>Document</title>
+</head>
+<body>
+    <h1 id="title">Survey Form</h1>
+    <p id= "description">Tell us about your favourite mode of transportation.</p>
+    <form id="survey-form">
+      <label id = "name-label" for="name">
+        Name: <input id = "name" type="text" placeholder = "Enter your Name"/>
+      </label>
+      <label id = "email-label" for="email">
+        Email: <input id = "email" type="email" placeholder = "Enter your Email"/>
+    </label>
+      <label id = "number-label" for="number">
+        Number: <input id = "number" min="0" max="10" type = "number" placeholder = "Enter your Number"/>
+    <label>
+        <br/>
+      <label id = "number-label" for="dropdown">
+        Vehicle:  <select id = "dropdown"> 
```

Comment:
I think using space-less equal signs is a better approach as it is considered a best practice and makes your code looks cleaner. Please try to use `<input id="number-label"` instead of `<input id="number-label `